openmc/data/data.py

```diff
@@ -11,106 +11,106 @@
-    'Sn115': 0.0034, 'Sn116': 0.1454, 'Sn117': 0.0768,
-    'Sn118': 0.2422, 'Sn119': 0.0859, 'Sn120': 0.3258,
-    'Sn122': 0.0463, 'Sn124': 0.0579, 'Sb121': 0.5721,
-    'Sb123': 0.4279, 'Te120': 0.0009, 'Te122': 0.0255,
-    'Te123': 0.0089, 'Te124': 0.0474, 'Te125': 0.0707,
-    'Te126': 0.1884, 'Te128': 0.3174, 'Te130': 0.3408,
-    'I127': 1.0, 'Xe124': 0.00095, 'Xe126': 0.00089,
-    'Xe128': 0.0191, 'Xe129': 0.26401, 'Xe130': 0.04071,
-    'Xe131': 0.21232, 'Xe132': 0.26909, 'Xe134': 0.10436,
-    'Xe136': 0.08857, 'Cs133': 1.0, 'Ba130': 0.0011,
-    'Ba132': 0.001, 'Ba134': 0.0242, 'Ba135': 0.0659,
-    'Ba136': 0.0785, 'Ba137': 0.1123, 'Ba138': 0.717,
-    'La138': 0.0008881, 'La139': 0.9991119, 'Ce136': 0.00186,
-    'Ce138': 0.00251, 'Ce140': 0.88449, 'Ce142': 0.11114,
-    'Pr141': 1.0, 'Nd142': 0.27153, 'Nd143': 0.12173,
-    'Nd144': 0.23798, 'Nd145': 0.08293, 'Nd146': 0.17189,
-    'Nd148': 0.05756, 'Nd150': 0.05638, 'Sm144': 0.0308,
-    'Sm147': 0.15, 'Sm148': 0.1125, 'Sm149': 0.1382,
-    'Sm150': 0.0737, 'Sm152': 0.2674, 'Sm154': 0.2274,
-    'Eu151': 0.4781, 'Eu153': 0.5219, 'Gd152': 0.002,
-    'Gd154': 0.0218, 'Gd155': 0.148, 'Gd156': 0.2047,
-    'Gd157': 0.1565, 'Gd158': 0.2484, 'Gd160': 0.2186,
-    'Tb159': 1.0, 'Dy156': 0.00056, 'Dy158': 0.00095,
-    'Dy160': 0.02329, 'Dy161': 0.18889, 'Dy162': 0.25475,
-    'Dy163': 0.24896, 'Dy164': 0.2826, 'Ho165': 1.0,
-    'Er162': 0.00139, 'Er164': 0.01601, 'Er166': 0.33503,
-    'Er167': 0.22869, 'Er168': 0.26978, 'Er170': 0.1491,
-    'Tm169': 1.0, 'Yb168': 0.00123, 'Yb170': 0.02982,
-    'Yb171': 0.14086, 'Yb172': 0.21686, 'Yb173': 0.16103,
-    'Yb174': 0.32025, 'Yb176': 0.12995, 'Lu175': 0.97401,
-    'Lu176': 0.02599, 'Hf174': 0.0016, 'Hf176': 0.0526,
-    'Hf177': 0.186, 'Hf178': 0.2728, 'Hf179': 0.1362,
-    'Hf180': 0.3508, 'Ta180': 0.0001201, 'Ta181': 0.9998799,
-    'W180': 0.0012, 'W182': 0.265, 'W183': 0.1431,
-    'W184': 0.3064, 'W186': 0.2843, 'Re185': 0.374,
-    'Re187': 0.626, 'Os184': 0.0002, 'Os186': 0.0159,
-    'Os187': 0.0196, 'Os188': 0.1324, 'Os189': 0.1615,
-    'Os190': 0.2626, 'Os192': 0.4078, 'Ir191': 0.373,
-    'Ir193': 0.627, 'Pt190': 0.00012, 'Pt192': 0.00782,
-    'Pt194': 0.32864, 'Pt195': 0.33775, 'Pt196': 0.25211,
-    'Pt198': 0.07356, 'Au197': 1.0, 'Hg196': 0.0015,
-    'Hg198': 0.1004, 'Hg199': 0.1694, 'Hg200': 0.2314,
-    'Hg201': 0.1317, 'Hg202': 0.2974, 'Hg204': 0.0682,
-    'Tl203': 0.29524, 'Tl205': 0.70476, 'Pb204': 0.014,
-    'Pb206': 0.241, 'Pb207': 0.221, 'Pb208': 0.524,
-    'Bi209': 1.0, 'Th230': 0.0002, 'Th232': 0.9998,
-    'Pa231': 1.0, 'U234': 0.000054, 'U235': 0.007204,
-    'U238': 0.992742
+  'H1': 0.9998550, 'H2': 0.0001450, 'He3': 0.000002,
+  'He4': 0.999998, 'Li6': 0.0485000, 'Li7': 0.9515000,
+  'Be9': 1.0, 'B10': 0.1965000, 'B11': 0.8035000,
+  'C12': 0.9894000, 'C13': 0.0106000, 'N14': 0.9962050,
+  'N15': 0.0037950, 'O16': 0.9975700, 'O17': 0.0003835,
+  'O18': 0.0020450, 'F19': 1.0, 'Ne20': 0.9048,
```

Comment:
I'm honestly not sure what to suggest for the 3-isotope interval cases. What is definitely not acceptable is to have the abundances sum to something other than 1. That leaves the following options:
1. Slightly modify a single value so that they sum to unity
2. Modify all values the same amount so that they sum to unity (will result in weird values)
3. Stick with the "best measurements" for these cases

I lean toward option 1 but would definitely like to hear others thoughts.